### ISSUE TITLE
Reg training ground truth for DAVIS dataset

### ISSUE THREAD

**Issue by InnovArul** [carrierlxk/COSNet#13]
Hi, Thanks for releasing the code. Really helpful!

Just wanted to clarify about the lines: https://github.com/carrierlxk/COSNet/blob/master/dataloaders/PairwiseImg_video.py#L201-L202

As I understand, DAVIS dataset contains labels (with non-zero indices) of multiple objects for same frame. And during your training, you are accumulating all the annotated objects with non-zero labels into a single class. Is that correct?

**Comment by carrierlxk** [comment]
Yes, if you want to use multiple instance labeled data, you need to convert them into a binary mask.
